src/community/ogcapi/ogcapi-core/pom.xml

```diff
@@ -172,6 +172,10 @@
       <version>${project.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
Oh dear, this is insanely bad. It means OGC APIs cannot be deployed anymore without a GUI in GeoServer.
Or, in other words, that GeoServer cannot be deployed anymore in secure environments where the GUI and REST APIs are banned (and need to be removed), as well as causing issues in environments were GUI is separate from services (e.g., GeoServer cloud). 

We have a web-ogcapi module fo a reason, please rework the design for this.